tests/pytest/conftest.py

```diff
@@ -112,6 +113,7 @@ def model_EnrollmentFlow_with_scope_and_claim(model_EnrollmentFlow, model_Claims
 @pytest.fixture
 def model_EnrollmentFlow_with_claims_scheme(model_EnrollmentFlow_with_scope_and_claim):
     model_EnrollmentFlow_with_scope_and_claim.claims_scheme_override = "scheme"
+    model_EnrollmentFlow_with_scope_and_claim.transit_agency_id = 1
```

Comment:
Same note here about using `transit_agency` instead of `transit_agency_id`. I guess this means you'll need to bring in the `model_TransitAgency` fixture to assign it.